app/code/core/Mage/Adminhtml/controllers/System/CurrencyController.php

```diff
@@ -78,7 +78,7 @@ public function fetchRatesAction()
             }
             $rates = $importModel->fetchRates();
             $errors = $importModel->getMessages();
-            if( sizeof($errors) > 0 ) {
+            if(count($errors)) {
```

Comment:
Space: `if (count($errors)) {`

Reply:
@kiatng I did not run phpCBF to not break other PRs. Want ed this simple as possible ...

Simple space errors can could be at the very very last end ...